.github/workflows/pr-ebpf.yaml

```diff
@@ -23,7 +23,29 @@ jobs:
             gofmt -s -d .
             exit 1
           fi
-      - name: Build
-        run: make all DOCKER=1
```

Comment:
the build step is now missing in the workflow isn't it?

Reply:
Now, It's working
- First build time (2m 53s): https://github.com/krol3/tracee/actions/runs/541588005
- Second time using cache(1m 51s): https://github.com/krol3/tracee/actions/runs/541596189
The actually time in the job pr-ebpf (1m 45s)

Reply:
is it actually building the executable thought? i don't think so since you removed this command, but maybe I'm missing something